ftw/workspace/foldercreator.py

```diff
@@ -0,0 +1,18 @@
+from ftw.workspace.content.folder import TabbedViewFolder
+from ftw.workspace.content.workspace import Workspace
+from ftw.zipextract.interfaces import IFolderCreator
+from ftw.zipextract.interfaces import _ObjectCreator
```

Comment:
What's the meaning of the leading `_` here?
Leading underscores usually mean that the thing is private and should not be used by other components 😉

Reply:
I meant it more as "should not be used as such" as `_ObjectCreator` would not work on its own. But in that case I'll rename it